modules/apps/commerce/README.markdown

```diff
@@ -45,7 +45,7 @@ Requirements: [Liferay Portal 7.1 GA1](https://github.com/liferay/liferay-portal
 3.  When the build is complete, run a full reindex of all search indexes. Go
     to--or refresh-- http://localhost:8080 in your browser. In the left-hand
     menu, go to *Control Panel* &rarr; *Configuration* &rarr; *Search* and
-    click *Execute* next to *Reindex all search indexes*.
+    click *Execute* next to *Reindex system and company indexes*.
```

Comment:
Should be "Virtual Instance" not "company" to follow the naming conventions of the UI.